OpenRA.Mods.Common/Lint/CheckActorReferences.cs

```diff
@@ -57,6 +57,9 @@ static void CheckTrait(Action<string> emitError, ActorInfo actorInfo, TraitInfo
 			FieldInfo fieldInfo, IReadOnlyDictionary<string, ActorInfo> dict, ActorReferenceAttribute attribute)
 		{
 			var values = LintExts.GetFieldValues(traitInfo, fieldInfo, attribute.DictionaryReference);
+			if (values == null)
```

Comment:
As I explained at length in Discord, this just silences the linter for potential issues that are then bound to crash ingame.
We need to either have to have a null check in the linter AND in every trait and weapon that has a collection marked with `ActorReferenceAttribute` (and any activity they may be passing them to), OR we can just agree to not leave such collections `null` (which we seem to be doing so far).

Reply:
I can reiterate as well that leaving references as null is completely valid choice. We do it very often. It sounds like you really want `<Nullable>enabled</Nullable>` in the project.